features/webxr-layers.yml

```
@@ -0,0 +1,71 @@
+name: WebXR Layers
+description: WebXR Layers offer a more efficient way to draw immersive content by providing layer types that are managed by the system compositor.
```

This description is very circular. "WebXR layers offer … layer types". I'm not sure this is useful for anyone not already familiar with it. I suspect something along these lines might be possible (with me BSing based on the spec's intro):

```suggestion
description: WebXR layer types, such as [mumble] or [mumble], are managed by the system compositor, to reduce latency or resampling.
```

---

```suggestion
description: WebXR layer types, such as `XRCylinderLayer`, `XRCylinderLayer`, `XREquirectLayer`, `XRProjectionLayer`  or `XRQuadLayer`, are managed by the system compositor, to reduce latency or resampling.
```